modules/rosa-delete-objects.adoc

```diff
@@ -0,0 +1,273 @@
+// Module included in the following assemblies:
+//
+// * cli_reference/rosa_cli/manage-objects-with-rosa.adoc
+
+[id="rosa-delete-objects{context}"]
+= Delete objects
+
+This section describes the `delete` commands for clusters and resources.
+
+[id="rosa-delete-admin_{context}"]
+== delete admin
+
+Deletes a cluster administrator from a specified cluster.
+
+.Syntax
+[source,terminal]
+----
+$ rosa delete admin --cluster=<clusterName>
+----
+
+.Arguments
+|===
+|Option |Definition
+
+|-c, --cluster
+|(Required) The name or ID (string) of the cluster to add to the identity provider (IDP).
+|===
+
+
+.Optional arguments inherited from parent commands
+|===
+|Option |Definition
+
+|-h, --help
+|Shows help for this command.
+
+|debug
+|Enables debug mode.
+
+|-i, --interactive
+|Enables interactive mode.
+
+|--profile
+|Uses a specific AWS profile (string) from your credentials file.
+
+|-v, --v level
+|Log level for V logs.
+|===
+
+.Example
+Delete a cluster administrator from a cluster named 'mycluster'.
+
+[source,terminal]
+----
+$ rosa delete admin --cluster=mycluster
+----
+
+[id="rosa-delete-cluster_{context}"]
+== delete cluster
+
+Deletes a cluster.
+
+.Syntax
+[source,terminal]
+----
+$ rosa delete cluster [ID | Name] [arguments]
+----
+
+.Arguments
+|===
+|Option |Definition
+
+|-c, --cluster
+|(Required) The name or ID (string) of the cluster to delete.
+
+|--watch
+|Watches the cluster uninstallation logs.
+|===
+
+.Optional arguments inherited from parent commands
+|===
+|Option |Definition
+
+|-h, --help
+|Shows help for this command.
+
+|debug
+|Enables debug mode.
+
+|-i, --interactive
+|Enables interactive mode.
+
+|--profile
+|Uses a specific AWS profile (string) from your credentials file.
+
+|-v, --v level
+|Log level for V logs.
+
+|-y, --yes
+|Automatically answers yes to confirm the operation.
+|===
+
+.Examples
+Delete a cluster named 'mycluster'.
+
+[source,terminal]
+----
+$ rosa delete cluster mycluster
+----
+
+Delete a cluster using the --cluster argument.
+
+[source,terminal]
+----
+$ rosa delete cluster --cluster=mycluster
+----
+
+[id="rosa-delete-idp_{context}"]
+== delete idp
+
+Deletes a specific identity provider (IDP) from a cluster.
+
+.Syntax
+[source,terminal]
+----
+$ rosa delete idp [ID | Name] [arguments]
+----
+
+.Arguments
+|===
+|Option |Definition
+
+|-c, --cluster
+|(Required) The name or ID (string) of the cluster from which the IDP will be deleted.
+|===
+
+.Optional arguments inherited from parent commands
+|===
+|Option |Definition
+
+|-h, --help
+|Shows help for this command.
+
+|debug
+|Enables debug mode.
+
+|-i, --interactive
+|Enables interactive mode.
+
+|--profile
+|Uses a specific AWS profile (string) from your credentials file.
+
+|-v, --v level
+|Log level for V logs.
+
+|-y, --yes
+|Automatically answers yes to confirm the operation.
+|===
+
+.Example
+Delete an identity provider named github-1 from a cluster named 'mycluster'.
```

Comment:
I'd put github-1 in backticks (and also mycluster as mentioned in another comment)

Reply:
Done.